README.md

```diff
@@ -33,170 +33,140 @@ The latest stable releases are (were) versions that are recommended for producti
-which language bindings it is going to build.
+Building FoundationDB requires at least 8GB of memory. More memory is needed when building in parallel. If the computer freezes or crashes, consider disabling parallelized build using `ninja -j1`.
 
+#### Build Using the Official Docker Image
 
-### Generating `compile_commands.json`
-
-CMake can build a compilation database for you. However, the default generated
-one is not too useful as it operates on the generated files. When running make,
-the build system will create another `compile_commands.json` file in the source
-directory. This can than be used for tools like
-[CCLS](https://github.com/MaskRay/ccls),
-[CQuery](https://github.com/cquery-project/cquery), etc. This way you can get
-code-completion and code navigation in flow. It is not yet perfect (it will show
-a few errors) but we are constantly working on improving the development experience.
-
-CMake will not produce a `compile_commands.json`, you must pass
-`-DCMAKE_EXPORT_COMPILE_COMMANDS=ON`.  This also enables the target
-`processed_compile_commands`, which rewrites `compile_commands.json` to
-describe the actor compiler source file, not the post-processed output files,
-and places the output file in the source directory.  This file should then be
-picked up automatically by any tooling.
-
-Note that if building inside of the `foundationdb/build` docker
-image, the resulting paths will still be incorrect and require manual fixing.
-One will wish to re-run `cmake` with `-DCMAKE_EXPORT_COMPILE_COMMANDS=OFF` to
-prevent it from reverting the manual changes.
+The official Docker image for building is [`foundationdb/build`](https://hub.docker.com/r/foundationdb/build), which includes all necessary dependencies. The Docker image definitions used by FoundationDB team members can be found in the [dedicated repository](https://github.com/FoundationDB/fdb-build-support). When building inside the container, it is required to use the compilers in the `toolset` subdirectory. To enable the `toolset`, import the environment variables by
 
-### Using IDEs
-
-CMake has built in support for a number of popular IDEs. However, because flow
-files are precompiled with the actor compiler, an IDE will not be very useful as
-a user will only be presented with the generated code - which is not what she
-wants to edit and get IDE features for.
-
-The good news is, that it is possible to generate project files for editing
-flow with a supported IDE. There is a CMake option called `OPEN_FOR_IDE` which
-will generate a project which can be opened in an IDE for editing. You won't be
-able to build this project, but you will be able to edit the files and get most
-edit and navigation features your IDE supports.
-
-For example, if you want to use Xcode to make changes to FoundationDB you can
-create an Xcode project with the following command:
-
-```sh
-cmake -G Xcode -DOPEN_FOR_IDE=ON <FDB_SOURCE_DIRECTORY>
+``` bash
```

Comment:
Should we add here the command to run the container with all the mount options? That would make it easier for people to get started. We might want to drop a note here about the new container tooling from Apple if building on MacOS (https://github.com/apple/container/tree/main).

Reply:
Sure, let me first learn how to use apple container.

Reply:
Yes, that comment wasn't meant to block this PR.